docs/src/tutorial/cython_tutorial.rst

```diff
@@ -165,6 +221,72 @@ which we can try out in the interactive interpreter as follows::
 See, it works! And if you're curious about how much work Cython has saved you,
 take a look at the C code generated for this module.
 
+
+It is always good to check where is the Python interaction in the code with the
+``annotate=True`` parameter in ``cythonize()``. Let's see:
+
+.. figure:: htmlreport.png
+
+The function declaration and return use the Python interpreter so it makes
+sense for those lines to be yellow. Same for the list comprehension because
```

Comment:
"makes sense ... to be yellow" - you didn't say what yellow is or means, so it makes no sense at all ;-)